### DISCUSSION TITLE
Linux distro independent packaging formats

### DISCUSSION THREAD

**Discussion by develcooking** [slint-ui/slint#5397] [Q&A] [answered]
Hi there,
is there a way to package a Slint + Rust application in a Appimage or a Flatpak?

**Comment by bjorn** [comment, marked as answer]
For [Raccoin](https://github.com/bjorn/raccoin) I have used [cargo-packager](https://crates.io/crates/cargo-packager) for this, which can make an AppImage but can also make release packages for macOS and Windows. Regarding Flatpak I found an [open discussion here](https://github.com/flatpak/flatpak-builder/issues/15), not sure if there's a generally acceptable solution yet.
